File: app/src/main/java/social/entourage/android/carousel/pages/CarouselPage1Fragment.java
package social.entourage.android.carousel.pages;


import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import butterknife.ButterKnife;
import social.entourage.android.R;

/**
 * Carousel page 1
 */
public class CarouselPage1Fragment extends Fragment {


    public CarouselPage1Fragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_carousel_page1, container, false);

        ButterKnife.bind(this, view);

        return view;
    }

}

File: app/src/main/java/social/entourage/android/carousel/pages/CarouselPage1Fragment.kt
package social.entourage.android.carousel.pages

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import social.entourage.android.R

/**
 * Carousel page 1
 */
class CarouselPage1Fragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? { // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_carousel_page1, container, false)
    }
}